india_banking_connector/utils.py

```
@@ -85,16 +85,34 @@ def decrypt(data, key=None):
 
 
 class ResponseObject:
-	def __init__(self, text, status_code):
+	def __init__(
+		self, text, status_code, method="POST", url=None, headers=None, body=None
```

Ruff (0.15.1)

[warning] Unused method argument: `method`

(ARG002)